blogposts/announcing-sourcegraph-3.8.md

```diff
@@ -0,0 +1,201 @@
+
+[**🔦 Quickly toggle the browser extension on/off**](#quickly-toggle-the-browser-extension-onoff)<br />
+
+[**📝 Changelog**](#changelog)<br />
+Every detail that changed in this release
+
+[**🎖️ Thank you**](#thank-you)<br />
+Sourcegraph couldn't be what it is without the community.
+
+</div>
+
+**Deploy or upgrade:** [Local](https://docs.sourcegraph.com/#quickstart-guide) | [AWS](https://github.com/sourcegraph/deploy-sourcegraph-aws) | [DigitalOcean](https://marketplace.digitalocean.com/apps/sourcegraph?action=deploy&refcode=48dfb3ccb51c) | [Kubernetes cluster](https://github.com/sourcegraph/deploy-sourcegraph)
+
+## Introducing Automation: first stages of campaign management
+
+TO DO: Include one of the demo videos here?
+
+<!--
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/{ID}?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/{ID}" target="_blank">View on Vimeo</a></p>
+</p>
+-->
+
+[Large scale code change Automation](https://about.sourcegraph.com/product/automation) is coming soon to Sourcegraph. With this functionality, you will be able to automate large-scale code changes to remove legacy code, fix critical security issues, and pay down tech debt. We have started our implementation with campaign management - the ability to track sets of changes (e.g. pull requests) to ensure they are rolled out throughout your organization.
+
+[See our prototype of Automation in action](https://about.sourcegraph.com/product/automation#see-it-in-action) and let us know if you are interested in learning more, or if Automation would benefit you and your company.
+
+## Beta release of fast and precise LSIF-based code intelligence
+
+TO DO: VIDEO?
+
+<!--
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/{ID}?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/{ID}" target="_blank">View on Vimeo</a></p>
+</p>
+-->
+
+Each Sourcegraph instance ships with basic code intelligence that provides jump to definition and find references for every language. Basic code intelligence works well for many of our customers, but some customers have been asking for more precise code intelligence. We previously used language servers to provide precise code intelligence, but they were complex to configure, required separate deployment, and were slow to initialize.
```

Comment:
This makes it sound like we no longer support language servers.

```suggestion
Each Sourcegraph instance ships with basic code intelligence that provides jump to definition and find references for every language. Basic code intelligence works well for many of our customers, but some customers have been asking for more precise code intelligence. Language servers provide precise code intelligence, but they are complex to configure, require separate deployment, and are slow to initialize, so we have added support for LSIF based code intelligence.
```

(then delete the first sentence of the next paragraph)